astra-4.0/Dockerfile

```diff
@@ -0,0 +1,105 @@
+FROM adoptopenjdk:8-jre-hotspot-bionic
```

Comment:
@stef1927 FYI I've amended this back to a jre image, same as the upstream OSS one (while I previously switched to a jdk image for a test).